tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingTestCase.java

```diff
@@ -236,17 +238,31 @@ public void testPushAllPackages() throws Exception {
         // Create first module
         balClient.runMain("add", new String[]{firstPackage}, envVariables, new String[]{}, new LogLeecher[]{},
                 projectPath.toString());
-    
-        Assert.assertTrue(Files.exists(projectPath.resolve("src").resolve(firstPackage)));
-        Assert.assertTrue(Files.isDirectory(projectPath.resolve("src").resolve(firstPackage)));
+
+        Path firstPackagePath = projectPath.resolve("src").resolve(firstPackage);
+
+        Assert.assertTrue(Files.exists(firstPackagePath));
+        Assert.assertTrue(Files.isDirectory(firstPackagePath));
+
+        PackerinaTestUtils.copy(Paths.get("src", "test", "resources", "packaging", "pushingModule", "main.bal"),
+                firstPackagePath.resolve("main.bal"));
+        PackerinaTestUtils.copy(Paths.get("src", "test", "resources", "packaging", "pushingModule", "main_test.bal"),
+                firstPackagePath.resolve("tests").resolve("main_test.bal"));
     
         // Create second module
         balClient.runMain("add", new String[]{secondPackage}, envVariables, new String[]{}, new LogLeecher[]{},
                 projectPath.toString());
-    
-        Assert.assertTrue(Files.exists(projectPath.resolve("src").resolve(secondPackage)));
-        Assert.assertTrue(Files.isDirectory(projectPath.resolve("src").resolve(secondPackage)));
-    
+
+        Path secondPackagePath = projectPath.resolve("src").resolve(secondPackage);
+
+        Assert.assertTrue(Files.exists(secondPackagePath));
+        Assert.assertTrue(Files.isDirectory(secondPackagePath));
```

Comment:
We can get rid of asserting whether file exists because Files.isDirectory(modulePath) will fail anyway if the file doesn't exist. WDYT?